src/core/net/srp_server.cpp

```diff
@@ -506,18 +506,17 @@ void Server::HandleNotifierEvents(Events aEvents)
 
 Error Server::PublishServerData(void)
 {
-    NetworkData::Service::SrpServer::ServerData serverData;
+    NetworkData::Service::DnsSrpUnicast::ServerData serverData(Get<Mle::Mle>().GetMeshLocal64(),
```

Comment:
[Beyond the scope of this PR]
Seems the SRP server only needs to support one format of Service TLV.
Should we switch to the Anycast TLV instead?

@wgtdkp @abtink

Reply:
To use to anycast model, I think we need to have a mechanism to discover and sync the current "seq number" if/when we have multiple SRP servers. Probably need SRP replication mechanism.